pkg/auth/handler/webapp/change_password.go

```diff
@@ -53,7 +54,8 @@ func (h *ForceChangePasswordHandler) GetData(r *http.Request, rw http.ResponseWr
 	viewmodels.Embed(data, baseViewModel)
 	viewmodels.Embed(data, passwordPolicyViewModel)
 	viewmodels.Embed(data, ChangePasswordViewModel{
-		Force: true,
+		Force:               true,
```

Comment:
Instead of adding PasswordRulesString to individual viewmodel, add it to passwordPolicyViewModel